third_party/nvidia/backend/driver.py

```diff
@@ -471,6 +507,7 @@ def __init__(self, src, metadata):
         self.launch = mod.launch
         self.global_scratch_size = metadata.global_scratch_size
         self.global_scratch_align = metadata.global_scratch_align
+        self.launch_cooperative_grid = 1 if metadata.launch_cooperative_grid == True else 0
```

Comment:
why is it `int` instead of bool?

Reply:
I was being a little paranoid about how bool would get passed through the C interop with `launch()`

I can change it back.